src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -97,37 +153,86 @@ private bool ApplicableOperatorIs(MethodSymbol candidate, CSharpSyntaxNode node,
             return true;
         }
 
-        private BoundPattern BindConstantPattern(
+        private BoundConstantPattern BindConstantPattern(
             ConstantPatternSyntax node,
             BoundExpression operand,
+            bool? operandIsNull,
```

Comment:
Would it be possible to calculate operandIsNull locally rather than forcing consumers of this API to do that?